src/components/forms/fragments/UnRegisteredBoatDetails.tsx

```diff
@@ -34,7 +34,11 @@ const UnRegisteredBoatDetailsFragment = ({
       <Row>
         {boatTypes && (
           <Col sm={4}>
-            <BoatType boatTypes={boatTypes} required={!fieldsNotRequired} />
+            <BoatType
+              boatTypes={boatTypes}
+              required={!fieldsNotRequired}
+              className="unregistered-boat-input"
```

Comment:
In the customer UI, we're following BEM naming convention, prefixed with `vene`
so this className would be something like: `vene-unregistered-boat__input`

Reply:
Will fix.